### PR TITLE
Fiv styling position

### PR DESCRIPTION
**Pull request by johnduffell**
## What does this change?
This is a myriad of styling fixes to get the banner from looking like a dog's dinner to pretty much matching the design! Thanks to @Ap0c for doing a lot of hand holding for me. 🥇 

I have also added in the read our story link with cmp code.
## Screenshots
---
Existing, AU/GB
Mobile
image
Tablet
image

980 (desktop)
image

1140 (leftcol)
image

1300 wide
image

---
Potential, AU/GB
mobile
image
tablet
image

desktop
image

leftcol
image

wide:
image

### CODE REVIEW

File: static/src/stylesheets/module/site-messages/_fiv-banner.scss [hunk 1]
```diff
@@ -122,7 +123,7 @@
 
 .fiv-banner__cta {
 
-    @include mq(tablet) {
+    @include mq(leftCol) {
```
Comment (katebee) [inline static/src/stylesheets/module/site-messages/_fiv-banner.scss]: You have two `@include mq(leftCol) {` 😄 
Could this block be combined with the one on line 138?

Comment (johnduffell) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: good point I'll fix

Comment (johnduffell) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: I didn't realise I asked commercial dev for a review, that was a slip, but thanks for taking a look! 😆

Comment (kelvin-chappell) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: @johnduffell You didn't.  See https://github.com/guardian/frontend/blob/master/.github/CODEOWNERS

**Comment by johnduffell** [comment]
I have updated the styling with Duarte's help and taken new screenshots in the description.

**Review by tomrf1 — APPROVED** [review]
looks good!

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 2]
```diff
@@ -39,44 +45,47 @@ export const acquisitionsBannerFivTemplate = (
     <div class="fiv-banner__container-${
         isPotential ? `potential` : `existing`
     }">
-        <div class="fiv-banner__text">
         ${
             isPotential
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: with more time and if this was going in longer term than just a week I'd be tempted to store the text in a data structure and map it into this html format, but at the moment it looks a bit hacky.

**Pull request merged**